src/app/services/tests/JoinerService.spec.ts

```diff
@@ -91,7 +91,7 @@ describe('JoinerService', () => {
         }));
         it('should delegate update to DAO', fakeAsync(async() => {
             await dao.set('joinerId', JoinerMocks.INITIAL);
-            service.observe('joinerId');
+            service.subscribeToChanges('joinerId', (doc: MGPOptional<Joiner>): void => {});
```

Comment:
Si tu veux GWT ceux où t'as fait que renommer, tu peux, après c'est un ticket plus gros in-fine que prévu donc te fait pas chier si un autre ticket te fait de l'oeil, très bon bumping sur ce tickets !

Reply:
not fixed ici, ça sera à l'occasion d'un passage sur tout le fichier de test